changelog/12314.improvement.md

```diff
@@ -0,0 +1 @@
+`SlotSet` events will be emitted when the value set by the custom action is the same as the existing value.
```

Comment:
I'd give more context that this case needs to be handled to allow the AugMemo Policy to function properly.

Reply:
Sure, I've added this context to the changelog